.github/workflows/test.yml

```diff
@@ -6,10 +6,21 @@ on:
 jobs:
   test:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python: ["3.6", "3.7", "3.8", "3.9"]
     steps:
       - uses: actions/checkout@v2
         with: { submodules: recursive }
+      - uses: conda-incubator/setup-miniconda@v2
+        with: { mamba-version: "*", channels: "conda-forge", channel-priority: true }
       - name: install dependencies
         shell: bash -l {0}
         run: |
-          echo Hello World.
```

Comment:
I had to add a dummy action directly to master for the CI to actually run. This is only a problem the very first time CI is added to a project on github.